test/test_max_storage_in_orthanc_raw.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python3
+
+#  Copyright (c) University College London Hospitals NHS Foundation Trust
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+"""
+We need to test that orthanc-raw is enforcing the configured maximum storage
+limit.  This script will upload sufficient instances to orthanc-raw that the
+maximum storage capacity will be reached and then confirm that the original test
+images are no longer stored.
+
+Polling to allow for orthanc processing time.
+"""
+import logging
+import tempfile
+from pathlib import Path
+from time import sleep
+
+import pytest
+import requests
+from decouple import config
+from pytest_pixl.dicom import write_volume
+
+SECONDS_WAIT = 5
+
+raw_instances_url = "http://localhost:{}/instances".format(config("ORTHANC_RAW_WEB_PORT"))
+
+
+@pytest.mark.usefixtures("_setup_pixl_cli")
+def test_max_storage_in_orthanc_raw():
```

Comment:
Can I confirm that it's this *entire file* (plus the `.sh` version in `test/scripts`) that we wanted to delete?

Reply:
yep!